etc/profile-a-l/lobster.profile

```diff
@@ -47,3 +47,5 @@ private-tmp
 
 # Redirect
 include mpv.profile
+
+read-write ${HOME}/.local/share/applications
```

Comment:
Or just use `ignore read-only ...`. Then it does not have to be after the `include`.